apps/fabric-website/homepage.htm

```diff
@@ -94,8 +111,8 @@
     var scripts = [appPath + entryPointFilename + '.min.js'];
 
     if (!window.React) {
-      var reactPath = '//cdnjs.cloudflare.com/ajax/libs/react/16.8.6/umd/react';
-      var reactDomPath = '//cdnjs.cloudflare.com/ajax/libs/react-dom/16.8.6/umd/react-dom';
+      var reactPath = '//cdnjs.cloudflare.com/ajax/libs/react/16.3.2/umd/react';
```

Comment:
@kenotron This is copied from the latest version of this file in the internal repo--not sure if reverting back to this old version was intentional?

Reply:
it isn't, i made this change in the homepage.htm before fabric 7 was released.

Reply:
although I think we should skip loading this altogether because the bundle now includes react I think...